### PR TITLE
Migrate CI to GitHub Actions

### PR DESCRIPTION
**Pull request by jathak**
We weren't previously testing on Windows, so this caught a bug with load paths that is also fixed in this PR

### CODE REVIEW

File: tool/grind.dart [hunk 1]
```diff
@@ -23,10 +23,9 @@ main(List<String> args) {
 
 @Task('Verify that the package is in a good state to release.')
 sanityCheckBeforeRelease() {
-  var travisTag = environment("TRAVIS_TAG");
-  if (travisTag != pkg.version.toString()) {
-    fail("TRAVIS_TAG $travisTag is different than pubspec version "
-        "${pkg.version}.");
+  var ref = environment("GITHUB_REF");
+  if (ref != "refs/tags/${pkg.version}") {
+    fail("TRAVIS_TAG $ref is different than pubspec version ${pkg.version}.");
```
Comment (nex3) [inline tool/grind.dart:28]:
```suggestion
    fail("GITHUB_REF $ref is different than pubspec version ${pkg.version}.");
```

**Review by nex3 — APPROVED** [review]
I'm guessing I'll need to add the various secrets to GH actions for this repo in order for it to be deployable?

**Comment by jathak** [comment]
> I'm guessing I'll need to add the various secrets to GH actions for this repo in order for it to be deployable?

Yes. Does that need to be done before merging this or after?

**Comment by nex3** [comment]
After is fine, it just needs to happen before the next deploy.

**Pull request merged**